src/fastapibootcamp/domain/models.py

```diff
@@ -0,0 +1,89 @@
+"""Models for the Astroplan-related app domain."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Any
+
+from astroplan import Observer as AstroplanObserver
+from astropy.coordinates import AltAz, Angle, SkyCoord
+from astropy.time import Time
+
+# We're re-exporting Observer from the app's domain models to simulate the idea
+# that the Astroplan Observer is the app's domain model (i.e., this is a
+# pecularity of this demo to make it look like we've coded Astroplan in the
+# app itself).
```

Comment:
I'm a touch confused by this comment, since the export here seems to be an `Observer` class defined in this module, not a class from astroplan.

Reply:
Oh yes. This was before I subclassed astroplan's observer to add some attributes. I'll update.